handbook/technical-overview/postholf/postholf-02-interface-skjalatilkynning.md

```diff
@@ -40,7 +40,7 @@ Response:
 
 ## DocumentIndex
 
-A document provider submits(registers) reference to documents. A reference consists of the name of the document, its identifier, owner kennitala (e. icelandic person/corp identity), along with other information. An organization may submit more than one reference that are referring to the same document, e.g. when a couple should see the same document. After the operation, the document becomes visible to the user of the web site(island.is).
+A document provider submits(registers) references to documents. A reference consists of the name of the document, its identifier, owner kennitala (e. icelandic person/corp identity), along with other information. A documentID and kennitala form a unique key pair. An organization may submit more than one reference that are referring to the same document, e.g. when a couple should see the same document. After the operation, the document becomes visible to the user of the web site(island.is).
```

Comment:
Fyi, we use "nationalId" instead of "kennitala" in all new Island.is code and APIs.

Fine to use "kennitala" assuming it matches the naming conventions in the current API.